app/views/dependencies/_dependencies.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if @dependencies.key?(scope) %>
+  <% if scope == "development" %>
+    <div id="<%=@latest_version.rubygem.name+scope%>"><span class="scope"><%= scope.titlecase %> :</span>
+    <div class="deps_scope deps_toggle">
+  <% else %>
+    <div id="<%=@latest_version.rubygem.name+scope%>"><span class="scope scope--expanded"><%= scope.titlecase %> :</span>
+    <div class="deps_scope">
+  <% end %>
+  <% @dependencies[scope].each do |name,version,req| %>
+    <ul class="deps">
+      <li>
+        <span>
```

Comment:
This span seems redundant.

Reply:
Actually I need this span because I'll be changing the contents of this span after it is clicked once. I'll be changing the content to just toggle the div's to follow. (So that it doesn't cause ajax requests anymore.

Reply:
I will be changing the contents of that span from .caret class (which I have now changed to deps_expanded-link) to the following for toggling.

![Screenshot from 2019-08-25 11-08-14](https://user-images.githubusercontent.com/16337625/63646036-b02eff00-c728-11e9-96f9-860862098aff.png)